### PR TITLE
Using the http-proxy-middleware to proxy

### PR DESCRIPTION
**Pull request by mofojed**
Alternative to Don's PR #172 . Needed to add the `ws: true` bit so websockets worked for the api as well.

### CODE REVIEW

File: packages/code-studio/package.json [hunk 1]
```diff
@@ -32,6 +32,7 @@
     "cross-env": "^7.0.2",
     "deep-equal": "^2.0.4",
     "fira": "github:mozilla/fira#4.202",
+    "http-proxy-middleware": "^2.0.1",
```
Comment (dsmmcken) [inline packages/code-studio/package.json:35]: dev-dependencies

Comment (mofojed) [reply packages/code-studio/package.json:35]: The instructions do specify it as a dependency...: https://create-react-app.dev/docs/proxying-api-requests-in-development/

I'll try

Comment (mofojed) [reply packages/code-studio/package.json]: Looks like it worked, good call

Comment (dsmmcken) [reply packages/code-studio/package.json]: CI did not like that 🤷, sorry.

**Review by dsmmcken — APPROVED** [review]

**Pull request merged**